docs/source/noise/column_noise.rst

```diff
@@ -4,18 +4,128 @@
 Column-based Noise
 ==================
 
+Column-based noise introduces errors in individual fields within a row of data.
```

Comment:
```suggestion
Column-based noise introduces errors in an individual cell within a row of data.
```

Reply:
As noted in the RT/SE check-in yesterday, it's probably confusing to refer to rows in the description of column noise, and especially here since we're not contrasting it with row-based noise. Perhaps we could say something like:

"Column-based noise operates on one column of data at a time, introducing errors to individual cells within the column."

Or something.